ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java

```diff
@@ -227,7 +227,7 @@ private void writeSubstations() throws IOException {
                         .writeCell(maxV)
                         .writeCell(faultNum)
                         .writeCell(actionNum)
-                        .writeCell(vl.getSubstation().getCountry().toString())
+                        .writeCell(vl.getSubstation().getCountry().map(Enum::toString).orElse("UNDEFINED"))
```

Comment:
In that case, I think an empty string is better. If you disagree, at least, consider to introduce a constant.